### PR TITLE
React OWIN article to ASP.NET Core rename

### PR DESCRIPTION
**Pull request by MaherJendoubi**

### CODE REVIEW

File: aspnet/fundamentals/owin.rst [hunk 1]
```diff
@@ -83,10 +83,10 @@ With this in place, all that's required to run an ASP.NET application using this
 
 When run, this command (equivalent to running ``dnx web`` from a command line) will search for a package called "NowinSample" that contains an implementation of ``IServerFactory``. If it finds one, it will initialize and start the server as detailed above. Learn more about the built-in ASP.NET :doc:`/fundamentals/servers`.
 
-Run ASP.NET 5 on an OWIN-based server and use its WebSockets support
+Run ASP.NET Core on an OWIN-based server and use its WebSockets support
 --------------------------------------------------------------------
```
Comment (danroth27) [inline aspnet/fundamentals/owin.rst]: The underline needs to be the full length of the section heading.

**Comment by danroth27** [comment]
@MaherJendoubi Could you please fix up the section heading on line 86? You can just add a commit and we will taking care of squashing the commits when we merge.

**Comment by MaherJendoubi** [comment]
what's wrong in line 86?

2016-02-10 21:50 GMT+01:00 Daniel Roth <notifications@github.com>:

**Comment by danroth27** [comment]
Well, the problem is really on line 87. The dashes need to extend the full length of the section heading (see http://www.sphinx-doc.org/en/stable/rest.html#sections).

**Comment by danroth27** [comment]
Perfect - thanks! 65f19ac56e23ceaa3f5e603fdad670159db4130b

**Pull request closed without merging**